internal/system/agent/executor/compose.go

```diff
@@ -119,37 +118,27 @@ func RunDockerComposeCommand(service string, dockerComposeService string) {
 
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		log.Fatal(fmt.Sprintf("Call to docker-compose up -d failed with %s\n", err))
+		log.Printf(fmt.Sprintf("Call to docker-compose up -d failed with %s\n", err))
```

Comment:
On statements like this, `err` should be `err.Error()`. The latter is a string which will replace `%s`. The former is not.